__main__ - optimizer_entrypoint(line:879) - !!! Skip optimizer !!! root_node_rps of all callgraph in all region is 0
__main__ - optimizer_entrypoint(line:879) - !!! Skip optimizer !!! root_node_rps of all callgraph in all region is 0
__main__ - optimizer_entrypoint(line:879) - !!! Skip optimizer !!! root_node_rps of all callgraph in all region is 0
__main__ - optimizer_entrypoint(line:879) - !!! Skip optimizer !!! root_node_rps of all callgraph in all region is 0
apscheduler.executors.default - run_job(line:131) - Job "optimizer_entrypoint (trigger: interval[0:00:01], next run at: 2024-04-05 06:19:02 UTC)" raised an exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/app/global_controller.py", line 1006, in optimizer_entrypoint
    assert False # TODO: not implemented yet
AssertionError
apscheduler.executors.default - run_job(line:131) - Job "optimizer_entrypoint (trigger: interval[0:00:01], next run at: 2024-04-05 06:19:07 UTC)" raised an exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/app/global_controller.py", line 1006, in optimizer_entrypoint
    assert False # TODO: not implemented yet
AssertionError
apscheduler.executors.default - run_job(line:131) - Job "optimizer_entrypoint (trigger: interval[0:00:01], next run at: 2024-04-05 06:19:21 UTC)" raised an exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/app/global_controller.py", line 1006, in optimizer_entrypoint
    assert False # TODO: not implemented yet
AssertionError
apscheduler.executors.default - run_job(line:131) - Job "optimizer_entrypoint (trigger: interval[0:00:01], next run at: 2024-04-05 06:19:45 UTC)" raised an exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/site-packages/apscheduler/executors/base.py", line 125, in run_job
    retval = job.func(*job.args, **job.kwargs)
  File "/app/global_controller.py", line 1006, in optimizer_entrypoint
    assert False # TODO: not implemented yet
AssertionError
